src/networking.c

```diff
@@ -1505,6 +1506,23 @@ void clientAcceptHandler(connection *conn) {
         }
     }
 
+    /* Auto-authenticate from cert_user field if set */
+    sds username = connGetPeerUsername(conn);
+    if (username != NULL) {
+        serverLog(LL_VERBOSE, "TLS: Attempting auto-auth using username = %s",
+                  server.hide_user_data_from_log ? "*redacted*" : username);
```

Comment:
This log message is redundant. It will be logged below in both cases: found and not-found user.

Reply:
Done remove